### DISCUSSION TITLE
Suggestion: Hide Npc's from Telekinetic Grab outside of MTA.

### DISCUSSION THREAD

**Discussion by Narwallus** [runelite/runelite#17973] [Ideas]
with the removal of the delay on telekinetic grab it is honestly quite fun to use, but during certain activities like slayer it can be a bit annoying when the npc's walk over the loot pile.
As far as I can recall the only place where being able to tele grab an npc does anything is in the telegrab room.

**Comment by joeamansfield** [comment]
I was looking for options to accomplish this, and I found this in the Custom Menu Swaps plugin:
"Some menu entries won't be swapped per jagex's rules (player menu entries, spell casts, blackjacking and construction)."
That said, they changed the rules about construction, so I'd expect to see them change the rule for spells to be changed as well if people asked for it.

**Reply by Narwallus** [reply]
oh that's interesting, thanks for letting me know :)
